great_expectations/datasource/data_connector/inferred_asset_sql_data_connector.py

```diff
@@ -8,7 +8,7 @@
 from great_expectations.datasource.data_connector.configured_asset_sql_data_connector import (
     ConfiguredAssetSqlDataConnector,
 )
-from great_expectations.execution_engine import ExecutionEngine
+from great_expectations.execution_engine import SqlAlchemyExecutionEngine
```

Comment:
It's not apparent to me why this can't be in a type checking block but, if I attempt to put it in one, python will die importing this file.

Reply:
There's no from future import annotations on this file

Reply:
Thank you!